lib/deploy/events/apiGateway/resources.js

```diff
@@ -6,65 +6,247 @@ const _ = require('lodash');
 module.exports = {
```

Comment:
Are those changes of this file for share API feature?
If so, could you remove those from this PR?

Reply:
Hi @horike37 thank you for review! Yes those changes are needed for mapping serveless API Gateway configuration to CloudFormation resource properties e.g.:

```yml
  apiGateway: # Optional API Gateway global config
    restApiId: n42gvt3whk # REST API resource ID. Default is generated by the framework
    restApiRootResourceId: 9f8qmgn5d6 # Root resource ID, represent as / path
    restApiResources: # List of existing resources that were created in the REST API. This is required or the stack will be conflicted
      '/playground': skvi34
```